### DISCUSSION TITLE
Multiple seed files for local development

### DISCUSSION THREAD

**Discussion by kylerummens** [supabase/supabase#13724] [Feature Requests]
It would be handy to be able to have multiple seed files in local development. Maybe instead of the seed file having to be called `seed.sql`, any file in the `/supabase` top-level directory with the `.sql` extension is considered a seed file?

Thoughts?

**Comment by yingw787** [comment]
I'd like something like Supabase recognizing seed*.sql so I can seed by data model :smile:

**Reply by matart15** [reply]
`seeds/*.sql` would be nice to manage

**Comment by matart15** [comment]
same : https://github.com/orgs/supabase/discussions/15646?
